src/net.h

```diff
@@ -687,16 +655,6 @@ class CNode
 
     void copyStats(CNodeStats &stats);
 
-    static bool IsWhitelistedRange(const CNetAddr &ip);
-    static void AddWhitelistedRange(const CSubNet &subnet);
-
-    // Network stats
-    static void RecordBytesRecv(uint64_t bytes);
-    static void RecordBytesSent(uint64_t bytes);
-
-    static uint64_t GetTotalBytesRecv();
-    static uint64_t GetTotalBytesSent();
-
     // resource deallocation on cleanup, called at node shutdown
     static void NetCleanup();
```

Comment:
It seems to me that `static void NetCleanup();` can be removed, being only used the `CConnman` implementation.

Reply:
Nicely spotted! Totally forgot to remove that from `CNode`...